src/api/filesystem.c

```diff
@@ -263,7 +263,8 @@ static int l_lovrFilesystemMount(lua_State* L) {
   const char* path = luaL_checkstring(L, 1);
   const char* mountpoint = luaL_optstring(L, 2, NULL);
   bool append = lua_isnoneornil(L, 3) ? 0 : lua_toboolean(L, 3);
-  lua_pushboolean(L, !lovrFilesystemMount(path, mountpoint, append));
+  const char* mountpointInner = luaL_optstring(L, 4, NULL);
```

Comment:
minor: I think this would be easier to understand if it was named "root" (it seems to also match up with the PhysFS terminology).